### DISCUSSION TITLE
Win 10 Version and Pre-requisites for WebView 2..?

### DISCUSSION THREAD

**Discussion by jbkalla** [MicrosoftEdge/WebView2Feedback#3872] [Q&A]
Hi!  I'm attempting to install Camtasia 2021, which requires WebView 2, but on one of our systems, WebView 2 won't install.  What are the prerequisites for WebView 2?  What minimum version of Windows or Windows frameworks do we need?  I believe that specific install is Win 10 v1809.

**Comment by champnic** [comment]
Hey @jbkalla - the WebView2 Runtime should be installable down to Windows 7, so Win10 1809 should be fine. How are you running the installer? Are you getting an error code?

**Comment by jbkalla** [comment]
Unfortunately, the app doesn't install and there's no error or dialog box or anything.  Event viewer doesn't show anything either.

**Comment by champnic** [comment]
Are you a dev on Camtasia looking to avoid a potential issue in the future, or just trying to get Camtasia installed and working on a machine?

If you just need to manually install the WebView2 runtime for Camtasia to work, you can do so by downloading and installing either the bootstrapper of the standalone installer here:
https://developer.microsoft.com/en-us/microsoft-edge/webview2/

**Comment by champnic** [comment]
To get installer logs, from an admin command prompt:
```
cd \
dir /s /b *edge*log
```
And get the logs files returned from that search. You can email me directly at champnic@microsoft.com.

(There may be some log files with `\Containers\` in the path - you don't need to send those)

**Comment by jbkalla** [comment]
Thanks, @champnic!  I've forwarded the command prompt info to our IT guy that's attempting to install it.
I'm not a dev, but we work on systems that don't have a connection to the Internet, which probably doesn't happen often for Camtasia very often.  We're just trying to get Camtasia installed and working.  We used the standalone installer, but it wouldn't install on one of our systems for some reason.  Once we get the logs, I'll let you know what they say.  I can't get you the actual log files, as they're on a system we have no way of getting files from.
I'll be able to get the log file to you in the next day or so.  It has to go through a downscan process that's pretty involved.

**Comment by champnic** [comment]
@jbkalla Sounds great, thanks John!
